app/build.gradle

```diff
@@ -56,6 +58,7 @@ android {
         versionCode 27
         versionName "1.3.0"
         buildConfigField "boolean", "DEBUG_MODE", "true"
+        buildConfigField "boolean", "CRASH_REPORTING_AVAILABLE", "false"
```

Comment:
"enabled"

Reply:
we had discussion about this. This controls if switch to turn it on is visible in settings, not if it is enabled

Reply:
oh ok thanks!